sql/sys_vars.cc

```diff
@@ -416,6 +416,16 @@ static Sys_var_enum Sys_vers_alter_history(
        SESSION_VAR(vers_alter_history), CMD_LINE(OPT_ARG),
        vers_alter_history_keywords, DEFAULT(VERS_ALTER_HISTORY_KEEP));
 
+static const char *vers_ident_mode_keywords[]= {"CURRENT", "HISTORICAL_EARLY",
+                                                "HISTORICAL", NULL};
+static Sys_var_enum Sys_vers_ident_mode(
+       "versioning_identifier_mode", "Versioning identifier selection mode. "
+       "CURRENT: current table column names to archive table;"
+       "HISTORICAL_EARLY: take earlier of a range of archive tables;"
```

Comment:
historical table and column names, earliest for range

Reply:
fixed